pkgs/development/libraries/libuev/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "libuev";
+  version = "2.4.0";
+  src = fetchFromGitHub ({
+    owner = "troglobit";
+    repo = pname;
```

Comment:
```suggestion
    repo = "libuev";
```

Reply:
why?

Reply:
Because this value will not change, therefore there is no need to create this dependency between the lines 4 and 8.

Nix is not a macro substitution language. There is no benefit in creating such dependency links around the code.